src/main/java/com/epam/aidial/core/controller/ResourceOperationController.java

```diff
@@ -13,25 +19,35 @@
 import com.epam.aidial.core.util.ProxyUtil;
 import io.vertx.core.Future;
 import io.vertx.core.Vertx;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServerResponse;
 import lombok.extern.slf4j.Slf4j;
 
+import java.util.Set;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
 @Slf4j
 public class ResourceOperationController {
 
+    private static final Set<ResourceType> SUBSCRIPTION_ALLOWED_TYPES = Set.of(
```

Comment:
files is not supported

Reply:
Disabled for now, but they will be in the next MR.